traveller/modules/conf/models.py

```diff
@@ -107,8 +106,8 @@ class Talk(PkModel):
         info={'choices': [('accepted', 'accepted'), ('pending', 'pending'), ('rejected', 'rejected')]})
 
     submitter_id = db.Column(db.Integer)
-    conf_id = db.Column(db.Integer, db.ForeignKey('conferences.id'),
-        nullable=False)
+    year_submitted = db.Column(db.String(4), nullable=False)
```

Comment:
Also why is year a string?

Reply:
I made it a string field because the I needed to give a length which is equal to 4. The integer field does not provide me with an option to add the length of the integer

Reply:
well np as validation goes code wise or in an sql routine at the very least.

Reply:
okay, I have reverted the PR so I will work on that. Thanks